catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/converter/MysqlExceptionConverter.java

```diff
@@ -15,21 +15,22 @@
 /** Exception converter to gravitino exception for MySQL. */
 public class MysqlExceptionConverter extends JdbcExceptionConverter {
 
+  @SuppressWarnings("FormatStringAnnotation")
```

Comment:
The string that is annotationed by `@FormatString` must be a final string or a compile time constant value. a variable is not allowed, such as 
```
new NoSuchEntityException(se.getMessage)
```

So I just ignore it by `@SuppressWarnings("FormatStringAnnotation")`